app/src/main/java/com/duycuong/weather/data/source/remote/LoadDataAsync.java

```diff
@@ -0,0 +1,76 @@
+package com.duycuong.weather.data.source.remote;
+
+import android.os.AsyncTask;
+
+import com.duycuong.weather.data.source.WeatherDataSource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
```

Comment:
ERROR: Unused import: java.nio.charset.StandardCharsets. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports